doc/nrf/gs_installing.rst

```diff
@@ -52,6 +52,7 @@ The installation process is different depending on your operating system.
          :dedent: 6
          :start-after: .. _install_dependencies_windows:
          :end-before: #. Close the window and open a new
+      Ensure that these dependencies are installed with their corresponding versions as specified in the :ref:`Required tools table <req_tools_table>`.
```

Comment:
Is there a reason to add this for Windows only?
How can I ensure that the correct versions are installed?

Reply:
These steps are rendered from zephyr and if we manually update it any updates on zephyr site will be lost.
That's the reason I have just added a instruction. I can put the same instruction in MacOS as well. Linux already has this instruction. Any other suggestions are welcomed  :).

Reply:
I have kept the same instruction in all the 3 Operation systems.